mlir/test/Dialect/OpenMP/ops-2.mlir

```diff
@@ -0,0 +1,74 @@
+// RUN: mlir-opt -verify-diagnostics %s | mlir-opt | FileCheck %s
```

Comment:
It seems all MLIR operations' tests are in `ops.mlir`.
Is it necessary to put these tests on a new file?
Also, `--verify-diagnostics` doesn't seem to be needed, as there are no `expected-*` lines.

Reply:
I moved the new test back to `ops.mlir`. I split it originally to make debugging test errors a bit easier.